### DISCUSSION TITLE
Why is juicefs data not deleted after deleting hive table using spark?

### DISCUSSION THREAD

**Discussion by XLPE** [juicedata/juicefs#4165] [Q&A] [answered]
I used the following ddl to create the table：

`CREATE TABLE IF NOT EXISTS test(
  name STRING,
  age INT
)
PARTITIONED BY (partition_col INT)
LOCATION 'jfs://bigjfs/test'`

After inserting a batch of data in bulk, execute it via spark-sql
`drop table test `
I find that the data is not deleted.
Even if I set trash-days to 0 and run the gc command, the data is still not deleted.

**Comment by zhijian-pro** [comment, marked as answer]
1. Keep a mount point that is always running
2. Check whether it is an external table
